app/build.gradle

```diff
@@ -24,7 +24,8 @@ android {
     }
     sourceSets {
         test.java.srcDirs += 'src/test/kotlin'
-        test.java.srcDirs += 'src/androidTest/kotlin'
+//        test.java.srcDirs += 'src/androidTest/kotlin'
```

Comment:
I should have removed it instead of commenting it out. 

@martintoften I believe it adds the tests under that path as regular unit-tests, running on the JVM instead of on an Android-device.